### DISCUSSION TITLE
Lottie: will text from a string be supported in 1.0?

### DISCUSSION THREAD

**Discussion by CoreMeltFx** [thorvg/thorvg#2867] [Q&A]
Eg using this line:
   "t": "Hello Now",

Attached is a complete example which renders as I'd expect in lottie player but does not display anything in the ThorVG viewer.
[lottie_embedtext_monospace.json](https://github.com/user-attachments/files/17395464/lottie_embedtext_monospace.json)

**Comment by hermet** [comment]
+ @tinyjin

**Comment by tinyjin** [comment]
Hey there @CoreMeltFx 

Font-based text is now supported in the ThorVG system with default font fallback.

When specifying a font name in a Text Document, please assign it to the "f" property using the "fName" value, instead of "fFamily".

<img width="852" height="840" alt="CleanShot 2025-10-01 at 12 24 42@2x" src="https://github.com/user-attachments/assets/7448fa53-65db-4032-9a1d-fea0ee1eb05f" />

With this change, the text should render correctly.

<img width="3012" height="1726" alt="CleanShot 2025-10-01 at 12 28 17@2x" src="https://github.com/user-attachments/assets/02441435-2ba5-4187-af56-a274db80f661" />
